integration-tests/python-enclave/signing/sample_private_key.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Any particular reason why we need a fixed signing key? If we can avoid it then it's one less file

Reply:
It is used when building the SGX manifest, what would be the alternative? Don't we do similar things elsewhere?

Reply:
We do, but that might be due to other reasons, like having a stable MRSIGNER value. Here, I'm not sure we need that now. In the `release` block in the enclave build.gradle you can specify `dummyKey`.

Reply:
Ok, great. Note: I was bitten by the enclave issue we spoke about, I need to clean the enclave in order for the change to be happening and to run the test

Reply:
DOne